### ISSUE TITLE
UI/UX feedback on spacing, more condensed dashoard, UI connections and more

### ISSUE THREAD

**Issue by LefterisJP** [rotki/rotki#3359]
## Abstract

User poolside left some feedback on the discord chat.

1. He argues that the current dashboard of rotki has a lot of waste of screen real estate and attached a screenshot of cryptowatch's portfolio overview on a single page. Access to all assets by holdings and exchanges and order infos etc!!!

![Screen_Shot_2021-08-22_at_11 14 09_AM](https://user-images.githubusercontent.com/1658405/130367067-bba609a6-b311-49cc-83c0-0396d0fa1762.jpg)

2. He also argues for interoperability between UI pages. So navigation between as many things as possile. His words:

![Screen_Shot_2021-08-22_at_11 43 17_AM](https://user-images.githubusercontent.com/1658405/130367097-286ca89b-b26a-4099-a1f0-5e0641e16e4c.jpg)

> I guess the thing I kind of miss is some sort of interoperabilty once you get to any point in Rotki. Let's say I land on a page like in that screenshot. I see numbers but it doesn't 'lead' me to another step or practical overview of sorts.  I am a very active trader (a combination of manual day trading and bots and hodling) I need to constantly juggle budgets/coins across several exchanges/accounts/dexes. So when I land on a page like this, I'd expect to be able to get to the next step of following a TRAIL, namely a) where are the holdings of this particular coin? b) what's my current portfolio value/% of it c) which other deals exchanges is this asset being used in d) trading/investment specific statistics: what's my PNL/ROI/Breakeven point for that coin/position?

3. For his comment of PNL/ROI breakeven point we have this issue we want to add to allow just that: https://github.com/rotki/rotki/issues/701 . But the comment on allowing all this to be interoperable and navigatable is imo really important and probably not that difficult.

## Task

Try to address as many of them as possible.

**Comment by kelsos** [comment]
@lukicenturi if you have any ideas for this, at least for the parts that are in the public part of the app you can give it a try/

**Comment by LefterisJP** [comment]
@lukicenturi and @kelsos which of these feedback have not been addressed and could have their own issue? I would like to close the issue but also make sure to keep whatever is actionable open in a separate issue ... if anything! If nothing we can close.

**Comment by kelsos** [comment]
@lukicenturi I am gonna close it, I think that at least some of the more specific suggestions have already been implemented. Do you see anything here that we might have missed?
